dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops

```diff
@@ -662,6 +662,14 @@ warning.variable.invalid=CWWKE0941W: The variable declaration {0} specified in s
 warning.variable.invalid.explanation=Only alphanumeric characters are allowed in variable names specified in the server.env file.
 warning.variable.invalid.useraction=Either modify the variable name to only use alphanumeric characters or move the definition of the variable to the bootstrap.properties file.
 
+error.zosProcStart.procedure.invalid=CWWKE0942E: The z/OS STC procedure {0} failed to start. The procedure name is invalid.
```

Comment:
I am assuming that there will be an ID review of these messages as well.  I think 'invalid' is one of those words that does not translate well to other languages.